pages/escape.md

```diff
@@ -0,0 +1,26 @@
+---
+title: ESCAPE
+---
+
+The [ESCAPE](https://projectescape.eu/) science cluster brings together communities and research infrastructures in the areas of astronomy and accelerator-based particle physics, specifically ESFRI facilities (SKAO, CTAO, KM3NeT, EST, ELT, HL-LHC, FAIR) and pan-European research infrastructures (CERN, ESO, JIVE). 
+
+Software applications for gathering and simulating the scientific data, in the order or exa-bytes, in ESCAPE span code bases of several million lines of source code and are written by researchers and engineers in various compiled (eg. C++, Cuda) and interpreted (eg. Python) languages. With experiments spanning over multiple decades the quality of the software developed and used is imperative for the success of the scientific endeavors in view of efficient computing and code maintainability.
```

Comment:
"the scientific data" -> "scientific data"
"*of* the order *of* exabytes" (no hyphen)
"eg." -> "e.g." x2

I'd say you should move "in ESCAPE" to before the sub-clause, to make the sentence scan better and you don't need to say "code bases" and "source code" as well - too many words...

"Software applications for gathering and simulating scientific data in ESCAPE, reaching up to exabytes, span code bases of millions of lines that are written by researchers and engineers in both compiled (e.g. C++, Cuda) and interpreted (e.g. Python) languages."

I would change "spanning over" to "existing for" and "imperative for" to "essential to". The last piece "in view of efficient computing and code maintainability" is somehow a sub-clause and feels awkward tacked onto the end. I would write it as:

"With experiments existing for multiple decades, high quality software, for efficient computing and code maintainability, is essential for the success of these scientific endeavours"

Generally less words because less is more 😁